OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs

```diff
@@ -94,6 +86,22 @@ public ActorSelectorLogic(Widget widget, World world, WorldRenderer worldRendere
 				ownersDropDown.ShowDropDown("LABEL_DROPDOWN_TEMPLATE", 270, owners, setupItem);
 			};
 
+			editorLayer.OnSpawnRemoved = () =>
+			{
+				if (editorLayer.Players.Players.Values.Any(p => p.Name.Equals(selectedOwner.Name)))
+					return;
+
+				var orderedOwners = editorLayer.Players.Players.Values.ToList();
+				orderedOwners.Add(selectedOwner);
+				orderedOwners.Sort((x, y) => String.Compare(x.Name, y.Name, StringComparison.Ordinal));
```

Comment:
> OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs:L96: [SA1121] Use the built-in type alias 'string' rather than String or System.String.

Use lower-case _s_tring.Compare.